src/tasks/withdraw/token_balances.ts

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+
+interface AddressInfoResponse {
+  tokens: Token[]
+}
+
+interface Token {
+  tokenInfo: TokenInfo;
+  balance: number;
+}
+
+interface TokenInfo {
+  address: string;
+  decimals: number;
+  price: TokenPrice;
+}
+
+interface TokenPrice {
+  rate: number;
+}
+
+export async function getTokensWithBalanceAbove(minValueUsd: string, settlementContract: string) {
+  const response = await axios.get(`https://api.ethplorer.io/getAddressInfo/${settlementContract}?apiKey=freekey`);
+  if (response.status !== 200) {
+    console.log(`Error getting tokens from ETHplorer ${response}`)
```

Comment:
If you want to keep it a log I would recommend using `console.error()`.
```suggestion
    console.error(`Error getting tokens from ETHplorer ${response}`)
```
But really this should throw an error IMO. Otherwise somebody might overlook the log and assume we don't have any tokens above that threshold in the contract.